catalog/plugin/catalog-plugin-metacardbackup/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPlugin.java

```diff
@@ -96,28 +88,28 @@
             return processRequest;
         }
 
-        if (StringUtils.isEmpty(outputDirectory)) {
+        if (CollectionUtils.isEmpty(storageBackupPlugins)) {
             throw new PluginExecutionException(
-                    "Unable to delete backup ingested metacard; no output directory specified.");
+                    "Unable to delete backup ingested metacard; no metacard backup storage provider configured.");
         }
 
         List<ProcessDeleteItem> processUpdateItems = processRequest.getProcessItems();
         for (ProcessDeleteItem processUpdateItem : processUpdateItems) {
             Metacard metacard = processUpdateItem.getMetacard();
-            deleteBackupIfPresent(metacard.getId());
+            try {
+                for (MetacardBackupStorageProvider storageProvider : storageBackupPlugins) {
```

Comment:
Do we want to do this for every instance of a storage provider?  Or do we want to be able to specify a list of them.  I may have 2 backup plugins, but I might want the first one to use storage provider a and b, and the other to use b and c.